docs/planning/backend/app/db/database.py

```
@@ -0,0 +1,57 @@
+"""
+Database configuration and session management.
+"""
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker, Session
+from contextlib import contextmanager
+from typing import Generator
```

Ruff (0.14.10)

Import from `collections.abc` instead: `Generator`

Import from `collections.abc`

(UP035)